R/tbl-df.r

```diff
@@ -2,6 +2,64 @@
 #' @exportClass tbl_df
 setOldClass(c("tbl_df", "tbl", "data.frame"))
 
+#' `tbl_df` class
+#'
+#' @description
+#' The `tbl_df` class is a subclass of [`data.frame`][base::data.frame()],
+#' created in order to have different default behaviour. The colloquial term
+#' "tibble" refers to a data frame that has the `tbl_df` class. Tibble is the
+#' central data structure for the set of packages known as the
+#' [tidyverse](https://www.tidyverse.org/packages/), including
+#' [dplyr](http://dplyr.tidyverse.org/),
+#' [ggplot2](http://ggplot2.tidyverse.org/),
+#' [tidyr](http://tidyr.tidyverse.org/), and
+#' [readr](http://readr.tidyverse.org/).
+#'
+#' The general ethos is that tibbles are lazy and surly: they do less and
+#' complain more than base [`data.frame`s][base::data.frame()]. This forces
```

Comment:
Link already in description:

```suggestion
#' complain more than base `data.frame`s. This forces
```

Reply:
With respect to links, I tend to think "more is more".